locales/de_DE.json

```diff
@@ -1995,6 +1995,8 @@
         "cancel-sell-past-amount": "Stornierter Verkauf von <0 />",
         "fail-buy-past-amount": "Kauf fehlgeschlagen <0 />",
         "fail-sell-past-amount": "Verkauf fehlgeschlagen <0 />",
+        
+        
```

Comment:
? Warum Leerzeilen ?

Das wird sowieso in JSON ignoriert, denke ich. Aber besser loeschen.